app/models/work_order.rb

```diff
@@ -47,4 +48,57 @@ def create_locked_set
     save!
   end
 
+  def send_to_lims
+    lims_url = product.catalogue.url
+    LimsClient::post(lims_url, lims_data)
+  end
+
+  def lims_data
+    material_ids = SetClient::Set.find_with_materials(set_uuid).first.materials.map{|m| m.id}
+    materials = MatconClient::Material.where("_id":{"$in" => material_ids}).result_set
```

Comment:
The materials & containers service will by default return only the first 25 results (which is an issue if the chosen Set has over 25 materials).

You can check if the `result_set` has more pages to fetch with `has_next?` (or use `is_last?`) and you can fetch the next page with `next`.